simplecal/simplecal

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Licensed under the terms of the GNU GENERAL PUBLIC LICENSE v2, or any later version.
+
+#
+# date pattern, more at 'man date'
+#
+
+DATE_PATTERN="%H:%M:%S %d.%m.%Y"
+
+#
+# command after clicking
+#
+
+CAL="gsimplecal"
```

Comment:
In case others prefer a different calender progrmam, use

    CAL="${BLOCK_INSTANCE:-gsimplecal}"

to allow the calendar to be chosen by setting the `instance` variable in the config.